accepted/0002-changelog.md

```diff
@@ -0,0 +1,53 @@
+# Changelog
+
+## Summary
+
+This RFC introduces the `changelog` command to `npm`.
+
+## Motivation
+
+A changelog file helps keep users of a library up to date on recent changes to
+that library. This allows them to better prepare for any bugs that might
+surface in their application from these changes.
+
+Since npm already supports including a changelog with the package by reserving
+three possible document names: `CHANGES`, `CHANGELOG`, and `HISTORY`
```

Comment:
["keep a changelog"](https://keepachangelog.com/en/1.0.0/) also suggests `NEWS` and `RELEASES`.

Reply:
The filenames listed here are the ones npm already guarantees will be published if they're present in the file. Adding new changelog filenames is, I think, out of scope for this RFC.